internal/service/router.go

```diff
@@ -22,27 +24,40 @@ func (s *service) router() chi.Router {
 			handlers.CtxLog(s.log),
 		),
 	)
-	r.Route("/integrations/CertIntegrator", func(r chi.Router) {
+	r.Route("/integrations/CertIntegrator-svc", func(r chi.Router) {
 		// configure endpoints here
 	})
 
 	return r
 }
 
 func (s *service) startListener(ctx context.Context, cfg config.Config) {
```

Comment:
It would be nice to separate API and core logic into to separated packages. Create an `api` folder where handlers, router etc. are placed and a core folder where an `indexer` folder is placed that contains listeners' logic.